README.md

```diff
@@ -54,6 +54,11 @@ Following are the implementations managed by third-parties adopting the standard
   * Machine-controller-manager, https://github.com/gardener/machine-controller-manager/tree/cluster-api
 
 ## Getting Started
+
+### Resources
+
+* GitBook: [kubernetes-sigs.github.io/cluster-api](https://kubernetes-sigs.github.io/cluster-api)
```

Comment:
I'm wondering if it would be best to hold off until the Netlify migration has been done for docs. @davidewatson thoughts?

Reply:
Is the URL going to change as part of that migration?

Reply:
It is, but I believe the URL is still to be determined. David is working on that effort and working with the test-infra group to coordinate the Netlify access and configs.

Reply:
OK,  it probably makes sense to wait if that's going to happen soon. I'm also happy to keep an eye on that work and propose another update to point to the new URL later when it's done.